project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.eed3si9n"      % "sbt-assembly"       % "2.1.1")
+addSbtPlugin("com.eed3si9n"      % "sbt-assembly"       % "1.2.0")
```

Comment:
Why we could not proceed with the update here?

I guess it may be worth to leave note in the code or in this PR so that the next person doing an upgrade in the future can have something to refer to.

Reply:
There is an issue with sbt-assemby. I added the comment with the tracking number